integration/spadev0/util.go

```diff
@@ -90,9 +90,9 @@ func AddSpadeTasks(ctx context.Context, requester string, replicasToTest map[int
 }
 
 var spadev0Metadata map[string]string = map[string]string{
-	"retrieve_type": "spade",
-	"retrieve_size": "1048576",
-	// todo: specify # of cids to test per layer of the tree TBD
+	"retrieve_type":  string(task.Spade),
+	"retrieve_size":  "1048576",
```

Comment:
why the extra space here

Reply:
it's the default `gofmt` formatter, which is part of the Go plugin for vscode - it's aligning all of the values to start at the vertical line (ie, because "traverse_depth" is a slightly longer string, it pushes them out an extra space)